backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -754,23 +754,11 @@ private void deleteUsedReleaseRelations(String projectId) throws SW360Exception
     //////////////////////
 
     public List<ProjectLink> getLinkedProjects(Project project, boolean deep, User user) {
-
-        final Map<String, Project> dbProjectMap;
-        if (deep){
-            dbProjectMap = ThriftUtils.getIdMap(repository.getAll());
-        } else {
-            dbProjectMap = preloadLinkedProjects(project, user);
```

Comment:
Please remove `preloadLinkedProjects(_,_)` as well in case it is not being used or not supposed to be used.

Reply:
removed